src/main/java/org/assertj/core/presentation/StandardRepresentation.java

```diff
@@ -138,6 +139,7 @@ public String toStringOf(Object object) {
     if (object instanceof Map<?, ?>) return toStringOf((Map<?, ?>) object);
     if (object instanceof Tuple) return toStringOf((Tuple) object);
     if (object instanceof MapEntry) return toStringOf((MapEntry<?, ?>) object);
+    if (object instanceof Method) { return ((Method) object).toGenericString(); }
```

Comment:
that a sufficient reason to me, especially if ones perform assertions on methods